build/buildRunTimeImages.sh

```diff
@@ -115,6 +115,11 @@ for dockerFile in $dockerFiles; do
         $labels \
         .
 
+    echo
+    echo "'$localImageTagName' image history:"
+    echo docker history $localImageTagName
```

Comment:
I added this because i am noticing even some runtime images' layers are a lot, so we might need to do some work over there too